README.md

```diff
@@ -1,3 +1,7 @@
+# *Changes implemented to the original JSQLTranspiler*
```

Comment:
We don't need this in the Readme, instead it will appear in the Changelog generated by Bit automatically

Reply:
sure.. commited by mistake.